setup.py

```diff
@@ -109,6 +110,13 @@ def find_version(*file_paths):
 
 version_string = "{}{}".format(sys.version_info[0], sys.version_info[1])
 
+_extra_deps = [
+    "tensorflow~=2.8.0",
+    "torch>=1.5.0, <=1.9.1, !=1.8.0",
+]
+
+extra_deps = {b: a for a, b in (re.findall(r"^(([^~!=<>]+)(?:[~!=<>].*)?$)", x)[0] for x in _extra_deps)}
+
```

Comment:
Those lines cannot affect to the actual dependency. Please refer to `EXTRA_REQUIRE`: https://github.com/openvinotoolkit/nncf/pull/1220/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R120-R139